src/Draggable/Plugins/Mirror/MirrorEvent/MirrorEvent.js

```diff
@@ -161,6 +161,44 @@ export class MirrorMoveEvent extends MirrorEvent {
   }
 }
 
+/**
+ * Mirror moved event
```

Comment:
How about add a tip 'Will be added in the next version' here and update it when release? Because our document is not updated when release version, but when pr merged.

Reply:
Yep, I updated it, please help to check it again 🎉